CMakeLists.txt

```diff
@@ -43,6 +43,7 @@ add_subdirectory (MAPL_cfio MAPL_cfio_r8)
 
 add_subdirectory (GMAO_pFIO)
 add_subdirectory (MAPL_Profiler)
+add_subdirectory (MAPL_ExceptionHandling)
```

Comment:
Let's call the new subdirectory "Shared"  And in any event no "MAPL_" on the subdirectory names.